### PR TITLE
Added and updated some IDs

### PR DESCRIPTION
**Pull request by RaphaelRoyerRivard**
I made a small mistake and reverted a commit, then recreated it. Will it cause problem?

### CODE REVIEW

**Comment by alkurbatov** [comment]
Yes, please, get rid of the merge commits. The history should be clear, otherwise it could be difficult to find out the reason of changes.

File: include/sc2api/sc2_typeenums.h [hunk 1]
```diff
@@ -456,6 +456,7 @@ namespace sc2 {
         EFFECT_VOIDRAYPRISMATICALIGNMENT = 2393,  // Target: None.
         EFFECT_WIDOWMINEATTACK = 2099,  // Target: Unit.
         EFFECT_YAMATOGUN = 401,   // Target: Unit.
+        GENERAL_MOVE = 3794,  // Target: Point.
```
Comment (alkurbatov) [inline include/sc2api/sc2_typeenums.h]: If you define GENERAL_MOVE it is ok to keep it as 3794, but you should use the old value for the MOVE command.

**Comment by RaphaelRoyerRivard** [comment]
I do not use that command, it's just that when debugging, I saw that
issuing a MOVE command sets an order to the unit with that GENERAL_MOVE
ability ID. Since it was not in the list, I had to search to find what that
ID meant. I thought it might be useful for someone else also.

**Comment by alkurbatov** [comment]
Here is a longer discussion which also gives a possible reason why we should keep old ID for the old command: #34

@mboedigh FYI

**Comment by RaphaelRoyerRivard** [comment]
I didn't remove the old MOVE command.

**Comment by alkurbatov** [comment]
I know, I mean: if you introduce GENERAL_MOVE, you should change id of the original MOVE command.

**Comment by RaphaelRoyerRivard** [comment]
Since the old MOVE command still works, I don't understand why we would change it. For what value should I change it?

**Comment by alkurbatov** [comment]
The old move command doesn't work in 4.10, but works in older version (with old ID). This is needed mostly for replay parsing and backward compatibility. See the discussion in the issue I linked above.
Feel free to drop this change, as I am going to add other generalized abilities.

File: include/sc2api/sc2_typeenums.h [hunk 2]
```diff
@@ -866,6 +867,7 @@ namespace sc2 {
         CARRYHARVESTABLEVESPENEGEYSERGAS = 273,
         CARRYHARVESTABLEVESPENEGEYSERGASPROTOSS = 274,
         CARRYHARVESTABLEVESPENEGEYSERGASZERG = 275,
+        RAVENSCRAMBLERMISSILE = 277,
```
Comment (alkurbatov) [inline include/sc2api/sc2_typeenums.h]: Please, add the new ID to the `BuffIDToName` function (`src/sc2api/sc2_typeenums.cc`).

**Pull request closed without merging**